src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+/**
+ * The class defines the type of machine - a bulldozer.
+ */
```

Comment:
For the future, you don't need to comment out methods, and classes, 
the code should tell us what is going on there